app/Http/Controllers/Bank/YodleeController.php

```diff
@@ -14,7 +14,7 @@
 use App\Helpers\Bank\Yodlee\Yodlee;
 use App\Http\Controllers\BaseController;
 use App\Http\Requests\Yodlee\YodleeAuthRequest;
-use App\Jobs\Bank\ProcessBankTransactions;
+use App\Jobs\Bank\ProcessBankTransactionsYodlee;
 use App\Models\BankIntegration;
 use Illuminate\Http\Request;
 
```

Comment:
Instead of renaming both props, we should only add prefixes for the new providers, so in this case lets leave everything without the yodlee_ prefix, but continue to prefix as nordigen

Reply:
I also touched the prop bank_integration_account_id for yodlee:
$table->renameColumn('bank_integration_account_id', 'bank_integration_yodlee_account_id');

should this also be reverted?

My Idea behind this was to structure both integrations independently, so there is no confusion, which parameter belongs to which one. Exspecialy with future dev.

Reply:
therefore I will revert the changes in a future commit?
I will come back here, if its done

Reply:
i reverted the column at the account table. do your want change of the Job Names?

"understood" could mean both to me🙈

Reply:
We should leave the yodlee props unchanged, only generate migratoins for the nordigen implementation

Reply:
@turbo124 do the jobnames also be reverted ((of yodlee)?